scripts/helpers.sh

```diff
@@ -13,6 +13,16 @@ is_osx() {
 	[ $(uname) == "Darwin" ]
 }
 
+is_chrome() {
+  chrome="/home/.shadow*"
+  path=$(df -Phk $HOME/Downloads | tail -1 | awk '{print $1}')
+  if [[ $path == $chrome ]]; then
+    return 0
+  else
+    return 1
+  fi
+}
+
```

Comment:
Hi Martin,
I personally use chromebook and in order to run debian/or other distro, it has to be installed on shadow path. B/c chromebook run a plain linux kernel (and runs ChromeOS on top of that), `uname` really does not give a clue. 
https://github.com/dnschneid/crouton gives a very common way to install ubuntu on chromebook without removing existing chromeos.